apps/desktop/src/app/sync/models.ts

```diff
@@ -19,8 +19,9 @@ export interface ISourceConnector {
   hasServerSideAuth: boolean;
   resumable: boolean;
   getParameters(): Observable<Field[]>;
+  handleParameters?(params: ConnectorParameters): void;
```

Comment:
shouldn't the `?` be just before the `:` ?
```handleParameters(params: ConnectorParameters)?: void;```

Reply:
If `?` is at the end then Typescript complains, I don't know why.